src/modules/courses/ui/views/course-booking-view.tsx

```
@@ -0,0 +1,144 @@
+              {isFull && <span className="text-red-500 ml-2">(Ausgebucht)</span>}
+              {!isFull && (
+                <span className="text-green-600 ml-2">({spotsAvailable} Plätze frei)</span>
+              )}
+            </p>
+          </div>
+
+          <div>
+            <h2 className="font-semibold text-gray-700 mb-2">Was Sie mitbringen sollten</h2>
+            <div
+              className="prose prose-sm max-w-none"
+              dangerouslySetInnerHTML={{ __html: course.description }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)